### ISSUE TITLE
Strip flicker

### ISSUE THREAD

**Issue by luxxl** [scottlawsonbc/audio-reactive-led-strip#105]
Today I tried to upgrade my strip and now have issue it started flickering and it appears while doing heavy job. Strip is 124 pixels long (i had 122 before) with additional 70 pixels wired with same data line from 27 pixel I also tried to disconnect it but same flicker occurs. I had it up and running perfectly fine and stable before assembling everything, but while assembling I soldered about 20 cm of data wire and it's like 1~1.2 meters long in total, all wires are 22 AWG I'm injecting power into 124 led's strip at 3 points (beginning, middle and at the end) and at one point on 70 led's. While investigating problem I noticed few things. First is when I turn on led's PSU first led turns green while all of them should turn off. Second thing is when I touch data line at first led (only first) flickering is gone and it's back as soon as I let it go. Third thing I was running @5.45vdc and when I increased voltage to something like 5.7 flickering sometimes disappear (depending on effects). By the way I noticed that when flickering appears led's shine much brighter than they should by program. It's definitely not PSU problem as it's rated at 75w and my led's only drawing about 5 amps of power. I'm wondering if it can be interference as data line runs kinda close to 5v line in some places. Thanks for help in advance.

**Comment by not-matt** [comment]
>  when I touch data line at first led (only first) flickering is gone and it's back as soon as I let it go
Maybe check that connection, and resolder if necessary?

**Comment by luxxl** [comment]
Connection by itself is solid there is no resistance on data line from rx pin to led strip I checked all of the connections for continuity before assembling because it requires to screw up a lot of screws and I kinda wanted to do it once I have some kind of logic booster chip and wondering if I should use it but it's strange that first led light up green after esp connecting to router and it's not always the case, sometimes it turns off. As i'm making contact with data line it completely stops flickering

**Comment by luxxl** [comment]
now i'm running @6vdc flicker is occurs only on energy when scale is maxed out kinda waiting for something to burn because of too high voltage :D

**Comment by charlieali** [comment]
Sounds like a grounding issue, check your grounds

**Comment by joeybab3** [comment]
did you end up fixing this? It sounds like it's not really a fault in the code so if you can, can you close this?

**Comment by luxxl** [comment]
Well, I think I managed to do some damage to controller because as I was investigating I ended up testing with new wires and even tried on new led strip but issue still occurred and I don't have another controller to test that atm. I didn't tough that it's code fault because it was working fine before I just noticed that when there is more intense instructions flicker occurred. Anyways waiting for new controller. Thanks for help ;)
